lib/super_good/solidus_taxjar/api.rb

```diff
@@ -25,7 +25,7 @@ def tax_for(order)
         taxjar_client.tax_for_order(ApiParams.order_params(order)).tap do |taxes|
           next unless SuperGood::SolidusTaxjar.logging_enabled
 
-          Rails.logger.info(
```

Comment:
These logs are very verbose - I'm not sold on enabling them by default! 

The fact that the `logging_enabled` variable was set to `false` by default makes me think that `logging_enabled` was intended to enable **debug** logs.

Could we hide these behind a debug logging flag or just log them with `debug` instead of `info` ?

Reply:
You are raising a good point here, should we just remove this in favour of the `debug` output which will contain the full response body? This seems redundant in the context of the logging we're adding in this PR 🤔

Reply:
If the logging added in this PR doesn't just log the response status and endpoint, then yeah, I think this would be redundant and removable!

If that's the case, could we add some tests for logging the response body?

Reply:
There is a test to match the body but it's a little bit hard to spot: https://github.com/SuperGoodSoft/solidus_taxjar/pull/175/files#diff-47df37a1fa490b55235d448f0a525a2f00e8d3e696588594f52fe20f7b0991feR26

We can't match on too much of the body because the response may change over time.

Reply:
We also decided to remove the inline log statement. It's completely redundant with request logging in place.

Reply:
Ahh cool! Sounds great!